azurelinuxagent/common/protocol/wire.py

```diff
@@ -232,6 +244,7 @@ def _build_health_report(incarnation, container_id, role_instance_id,
            u"<GoalStateIncarnation>{0}</GoalStateIncarnation>"
            u"<Container>"
            u"<ContainerId>{1}</ContainerId>"
+           u"<Users>{5}</Users>"
```

Comment:
I am not aware of any spec which describes this document - how do you know what is supported here?

Reply:
Removed